Fix #575 (FocusGained / FocusLost functionality)

extr0py opened the pull request:
- Listen to `BrowserWindow`'s `blur` and `focus` events
- Add an `executeAutoCommand` method to `NeovimInstance`
- Call `executeAutoCommand` on `blur`/`focus`

Fixes #575

cyansprite commented:
Do we redraw on focus gain?

Whenever I resize my window it shows a blank whatever background color is (test on windows of latest build)

extr0py commented:
@cyansprite - sorry, just saw your comment here!

We do not explicitly redraw on `FocusGained` - we might though if there is something in an autocmd that causes Neovim to ask for a redraw (like printing a message, changing line number format, etc).

Resize would likely be a separate issue - this PR had some changes related to resize: https://github.com/extr0py/oni/pull/557

When you say latest build, is it the latest __master__ or the latest downloadable build in the `Releases` tab? If its on __master__, feel free to open up an issue for it! Otherwise, there have been quite a few changes since the last binary release, so it might be fixed in master now.

cyansprite commented:
@extr0py 
Sorry, I meant building the latest source.
I will test it in a bit and get back to you, whether to open an issue or not.
